frontend/components/search/util/AutocompleteField.vue

```diff
@@ -16,11 +16,13 @@
     <template #message="{ message }">
       <v-tooltip max-width="40%" bottom open-delay="200">
         <template #activator="{ on }">
-          <!-- eslint-disable-next-line vue/no-v-html -->
-          <span v-on="on" v-html="message && message.length &lt; hintMaxWidth ? message : message.substring(0, hintMaxWidth) + '...'" />
+          <span v-on="on">
```

Comment:
We need to allow html code in hint as well, for example tags in https://github.com/PhiloBiblon/philobiblon-ui/blob/master/frontend/lang/en.js#L71